frontend/src/app/common/forms/forms.ts

```diff
@@ -89,7 +89,7 @@ type InitialValueOrInitialValueAndDisabled<T> =
  * For an example, see comment on the FormGroup wrapper class.
  */
 export class FormControl<
-  T extends string | number | File
+  T extends string | number | File | []
```

Comment:
Yeah, that is probably because the typsafe implementation currently cannot handle arrays on FormControls. This change will become irrelevant once we have found a general approach for handling arrays in FormControls.

Reply:
Hey Martin this was the work around code I did for to proceed with. Will be removed once your changes are merged.